tools/distpartitioning/data_shuffle.py

```diff
@@ -210,9 +210,14 @@ def exchange_node_features(rank, world_size, node_feature_tids, ntype_gnid_map,
     id_lookup : instance of class DistLookupService
        Distributed lookup service used to map global-nids to respective partition-ids and 
        shuffle-global-nids
-    node_feautres: dicitonary
-        dictionry where node_features are stored and this information is read from the appropriate
+    feautre_data: dicitonary
```

Comment:
typo: should be `feature_data`

Reply:
Thank you for pointing it out. Surely this will improve the overall readability of the code base. One step at a time. :-)